haystack/nodes/reader/table.py

```diff
@@ -371,30 +381,60 @@ def _aggregate_answers(agg_operator: Literal["COUNT", "SUM", "AVERAGE"], answer_
-        return results
+    def postprocess(self, model_outputs):
+        inputs = model_outputs["model_inputs"]
+        table = model_outputs["table"]
+        string_table = table.astype(str)
+        outputs = model_outputs["outputs"]
+        if self.type == "tapas":
+            if self.aggregate:
+                logits, logits_agg = outputs[:2]
+                copy_logits = logits.clone()
+                predictions = self.tokenizer.convert_logits_to_predictions(
+                    inputs, copy_logits, logits_agg, cell_classification_threshold=0.5
+                )
+                answer_coordinates_batch, agg_predictions = predictions
+                aggregators = {i: self.model.config.aggregation_labels[pred] for i, pred in enumerate(agg_predictions)}
+            else:
+                logits = outputs[0]
+                copy_logits = logits.clone()
+                predictions = self.tokenizer.convert_logits_to_predictions(
+                    inputs, copy_logits, cell_classification_threshold=0.5
+                )
+                answer_coordinates_batch = predictions[0]
+                aggregators = {}
+            answer_scores = self._calculate_answer_score(logits, inputs, answer_coordinates_batch)
+            answers = []
+            for index, ans_coordinates_per_table in enumerate(answer_coordinates_batch):
+                if len(ans_coordinates_per_table) > 0:
+                    cells = [table.iat[coordinate] for coordinate in ans_coordinates_per_table]
+                    aggregator = aggregators.get(index, "")  # type: ignore
+                    if aggregator == "NONE":
+                        answer_str = ", ".join(cells)
+                    else:
+                        answer_str = self._aggregate_answers(aggregator, cells)
+                    answer_offsets = _calculate_answer_offsets(ans_coordinates_per_table, string_table)
+                    current_score = answer_scores[index]
+                    answer = Answer(
+                        answer=answer_str,
+                        type="extractive",
+                        score=current_score,
+                        context=string_table,
+                        offsets_in_document=answer_offsets,
+                        offsets_in_context=answer_offsets,
+                        meta={
+                            "aggregation_operator": aggregator,
+                            "answer_cells": [string_table.iat[coordinate] for coordinate in ans_coordinates_per_table],
+                        },
+                    )
+                    answers.append(answer)
+                else:
+                    # If there is no answer then we use None to keep track of it.
+                    answers.append(None)
+        else:
+            raise NotImplementedError("Only TAPAS models are supported")
 
-    def predict_batch(self, queries: List[str], documents: List[List[Document]], top_k: int):
```

Comment:
This function was not removed, but relocated to higher-up in the file. It is still a member of the `_TapasEncoder` class.